### PR TITLE
Misc. typos follow up

### PR DESCRIPTION
**Pull request by luzpaz**
Follow up to previous commit #3462

### CODE REVIEW

File: libmscore/sym.cpp [hunk 1]
```diff
@@ -5035,8 +5035,8 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
       "Mute on",
       "Overpressure, down bow",
       "Overpressure, no bow direction",
-      "Overpressure possibile, down bow",
-      "Overpressure possibile, up bow",
+      "Overpressure possible, down bow",
+      "Overpressure possible, up bow",
```
Comment (IsaacWeiss) [inline libmscore/sym.cpp]: Let these stand. Italian.

File: libmscore/chordline.cpp [hunk 2]
```diff
@@ -365,7 +365,7 @@ void ChordLine::updateGrips(EditData& ed) const
             else if (_chordLineType == ChordLineType::PLOP)
                    ed.grip[0].translate(QPointF(-offset, -offset));
 
-            // translate on the length and height - stops the grips from goint past boundries of slide
+            // translate on the length and height - stops the grips from goining past boundaries of slide
```
Comment (IsaacWeiss) [inline libmscore/chordline.cpp]: "goining"

**Comment by luzpaz** [comment]
Revision pushed.

File: libmscore/dsp.h [hunk 3]
```diff
@@ -64,7 +64,7 @@ class Dsp {
             }
       virtual void cpy(float* dst, float* src, unsigned n) {
 #if defined(ARCH_X86) || defined(ARCH_X86_64)
-            register unsinged long int dummy;
+            register unsigned long int dummy;
```
Comment (Jojo-Schmitz) [inline libmscore/dsp.h]: This part must have been left uncompiled since quite a while

File: libmscore/sym.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply libmscore/sym.cpp]: Also a string we inherited from SMuFL, so can't change lightly

**Comment by luzpaz** [comment]
Rebased

**Comment by Jojo-Schmitz** [comment]
Better squash those commits into just one

**Comment by lasconic** [comment]
See http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by luzpaz** [comment]
Squashed and ready for merge

**Comment by lasconic** [comment]
Do you have more in store? I would prefer to have all of them in a single commit.

**Comment by luzpaz** [comment]
> Do you have more in store? I would prefer to have all of them in a single commit. 

@lasconic should be good to go :+1:

**Pull request merged**